### ISSUE TITLE
[N64] Quake and Quake II - Huge Input delay (almost a sec)

### ISSUE THREAD

**Issue by Matsilagi** [higan-emu/ares#1291]
**Describe the issue**
Quake series on the N64 emulators seems to have a notorious delay between button presses and the input taking effect. This does not happen in the console (Infact, Quake series are one of the snappier FPS games on the console)
To test, just load any of the ROMs and see for yourself!

**Rom Information**
Game name:              Quake
Product code:           NQKE0
Entry point:            [GOOD] 80000400
Byte order grade:       [GOOD] 1234
PI timings grade:       [GOOD] 80371240
File size grade:        [GOOD] 0xc00000 bytes
CRC grade:              [GOOD] 9f5bf79c d2fe08a0
IPL3 grade:             [GOOD] 6102 or 7101

```
file: Quake (USA).z64
sha256: ce43bcb4c11975f0e1720d7617c05bf23f420b1f76ab115ff52ba7344594a4bc
```

Reported only Quake 1 but Quake 2 should suffer from the same issue.

**Comment by invertego** [comment]
I recorded some videos with my phone, and I'm seeing very similar end-to-end latency (counting the frames from controller press to display update) in ares on my PC and on real N64 hardware connected to the same monitor via a Retrotink 2X-Pro.

For both setups, I'm seeing 167-200 ms latency at the start of the first mission in Quake. Latency for inputs on the main menu is even higher: 267-300 ms. As a point of comparison, I'm observing 67 ms of latency with Super Mario 64.

Based on these numbers I'm comfortable saying that Quake has observable input latency on real hardware, though it may feel better on a CRT. Are you able to measure the latency you're observing with ares on your own setup? Latency under emulation can depend on a number of factors (e.g. input device, operating system, drivers, emulator settings, monitor and display settings).

I'm mainly curious if you're literally observing "almost a second" of latency or if it simply feels that bad, because that's roughly 5x what I have measured, and if true that would imply something (unrelated to the emulation itself) has gone quite wrong.
